modules/viewing-resource-logs-cli-console.adoc

```diff
@@ -7,11 +7,11 @@
 [id="viewing-resource-logs-cli-console_{context}"]
 = Viewing resource logs
 
-You can view the log for various resources in the {first-oc} and web console. Logs read from the tail, or end, of the log.
+You can view the log for various resources in the {oc-first} and web console. Logs read from the tail, or end, of the log.
 
 .Prerequisites
 
-* Access to the {first-oc}.
+* Access to the {oc-first}.
```

Comment:
Given that you already introduce `OpenShift CLI (oc)` with `{oc-first}` in line 10, I do not think you need `{oc-first}` at line 14. Do we have an attribute that can be used for subquent references to `oc`?

Reply:
Pretty sure that's the only one